tests/rptest/redpanda_cloud_tests/omb_validation_test.py

```diff
@@ -0,0 +1,434 @@
+# Copyright 2023 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import math
+from time import sleep
+
+from rptest.services.cluster import cluster
+from rptest.tests.redpanda_test import RedpandaTest
+from ducktape.tests.test import TestContext
+from rptest.services.producer_swarm import ProducerSwarm
+from rptest.clients.rpk import RpkTool
+from rptest.services.redpanda_cloud import CloudTierName, ProductInfo, get_config_profile_name
+from rptest.services.redpanda import (SISettings, RedpandaServiceCloud)
+from rptest.services.openmessaging_benchmark import OpenMessagingBenchmark
+from rptest.services.openmessaging_benchmark_configs import \
+    OMBSampleConfigurations
+from tests.rptest.services.machinetype import get_machine_info
```

Comment:
I had to change this to `from rptest.services.machinetype import get_machine_info` to get it to run.

Reply:
Changing it now, thanks!